meta.json

```
@@ -30,6 +30,12 @@
         "short_desc": "來源：Trainers' Legend G (tinyurl.com/TLGCredits), 官方. 貢獻者：yotv2000tw (+維護者), seanpai96, asps76482, _akito@ytb, project90418512@twt, …"
     },
     {
+        "name": "UmaChimiID ( Indonesian + Local )",
+        "index": "https://raw.githubusercontent.com/Xmatography/umachimi-id/release/index.json",
+        "short_desc": "Translate Bahasa Indonesia + Bahasa Lokal ( slang ) Contributor : Xmatography , Vin , MisuMasu , … "
+    }
```

```suggestion
    },
```
Missing comma. This won't matter if you replace the other ID repo, but be careful to deliver valid JSON!